### PR TITLE
Add fox airdrop

### LINKED ISSUE #3275: Add shapeshift airdrop
https://fox.shapeshift.com/airdrop

### PR DESCRIPTION
**Pull request by yabirgb**
Closes #3275

## Checklist

- [x] The PR modified the frontend, and updated the [user guide](https://github.com/rotki/rotki/blob/develop/docs/usage_guide.rst) to reflect the changes.

### CODE REVIEW

**Comment by yabirgb** [comment]
Screenshot for reference 

image

File: rotkehlchen/constants/assets.py [hunk 1]
```diff
@@ -2327,3 +2327,15 @@
     protocol=None,
 )
 CONSTANT_ASSETS.append(A_RSPT)
+A_FOX = EthereumToken.initialize(
```
Comment (LefterisJP) [inline rotkehlchen/constants/assets.py:2330]: You forgot how we do this here. We never edit this file directly. You need to edit the template in the same file and then run `python tools/scripts/generate_constant_assets.py`

File: frontend/app/src/assets/images/airdrops/FOX_Icon_white.svg [hunk 2]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (LefterisJP) [inline frontend/app/src/assets/images/airdrops/FOX_Icon_white.svg:1]: You added two svgs but only one is used. Why?

**Review by LefterisJP — COMMENTED** [review]
Two small comments

File: frontend/app/src/assets/images/airdrops/FOX_Icon_white.svg (hunk 2 above)
Comment (yabirgb) [reply frontend/app/src/assets/images/airdrops/FOX_Icon_white.svg:1]: ShapeShift provides two icons, one for light theme and another for dark. @kelsos might find them useful to customize the behaviour and improve the UI

Comment (LefterisJP) [reply frontend/app/src/assets/images/airdrops/FOX_Icon_white.svg:1]: Okay but if he does not we shuld remove the extra svg. No need to make the final binary bigger without actually using stuff.

File: rotkehlchen/constants/assets.py [hunk 3]
```diff
@@ -819,7 +820,7 @@
     swapped_for=None,
     coingecko='aave-mkr',
     cryptocompare='MKR',
-    protocol=None,
+    protocol='aave',
```
Comment (LefterisJP) [inline rotkehlchen/constants/assets.py:823]: Hah nice side-effect of the script, right?!!?!?

File: rotkehlchen/tests/unit/test_ethereum_airdrops.py [hunk 4]
```diff
@@ -16,7 +16,7 @@ def test_check_airdrops(ethereum_accounts, data_dir):
 
     # Test data is returned for the address correctly
     assert len(data) == 2
-    assert len(data[TEST_ADDR1]) == 2
+    assert len(data[TEST_ADDR1]) == 3
```
Comment (LefterisJP) [inline rotkehlchen/tests/unit/test_ethereum_airdrops.py:19]: Lol that means I can claim the airdrop? TEST_ADDR1 is mine.

**Review by LefterisJP — APPROVED** [review]
LGTM, but **do not** merge yet. I am not happy about adding an unused file in the repo. Please rebase the PR to remove it and when kelsos implements the logic to switch between icons then he or your can add it.

**Comment by yabirgb** [comment]
@LefterisJP okey will do when I get home. I added it because is small but we can add it when we actually use it

**Review by LefterisJP — APPROVED** [review]
LGTM.

Failing test is a 502 from public kusama node

```
ERROR rotkehlchen.chain.substrate.manager: Kusama could not connect to node at endpoint: https://kusama.api.onfinality.io/public-https. Connection error: RPC request failed with HTTP status code 502.
```

merging

**Pull request merged**